src/main/java/com/playground/playground/GraphSystemController.java

```diff
@@ -0,0 +1,44 @@
+package com.playground.playground;
+
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.chart.NumberAxis;
+import javafx.scene.chart.ScatterChart;
+import javafx.scene.chart.XYChart;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+
+public class GraphSystemController implements Initializable {
+    @FXML
+    private NumberAxis yAxis;
+    @FXML
+    private NumberAxis xAxis;
+    @FXML
+    private ScatterChart neuralNetwork;
+
+    /**
+     * Construct the graph using the neural network.
+     * @param location
+     * The location used to resolve relative paths for the root object, or
+     * {@code null} if the location is not known.
+     *
+     * @param resources
+     * The resources used to localize the root object, or {@code null} if
+     * the root object was not localized.
+     */
+    public void initialize(URL location, ResourceBundle resources) {
+//        XYChart.Series<String, Double> seriesHigh = new XYChart.Series<>();
+//        seriesHigh.getData().add(new XYChart.Data<>("SubLabel1", 20.9));
+//        seriesHigh.getData().add(new XYChart.Data<>("SubLabel2", 30.9));
+//        seriesHigh.getData().add(new XYChart.Data<>("SubLabel3", 40.9));
+//
+//        XYChart.Series<String, Double> seriesLow = new XYChart.Series<>();
+//        seriesLow.setName("Label2");
```

Comment:
This function seems to be unimplemented, was it intended to be so? If so, could you please remove the commented code.

Reply:
The code is there for reference when implementing the graphing feature with the front end implementation.